siriuspy/siriuspy/orbintlk/csdev.py

```diff
@@ -53,7 +54,25 @@ class Const(_csdev.Const):
         ('IntlkCtrlRepeatTime-SP', 0),
         ('IntlkEvtIn0-SP', 117),
         ('IntlkEvtOut-SP', 124),
+        ('RxEnbl-SP', 0b00011110),
         )
+    FOUTS_CONFIGS = {
+        2: (
```

Comment:
why not condense this tuple in one line?

Reply:
I prefer to keep this format for minimizing future diffs.
But I can condense in one line.

Reply:
done!